### PR TITLE
Changes from UX assurance that couldn't be done with just styling

### PR DESCRIPTION
**Pull request by jachurchill**
*Issue #:* /bcgov/entity#6378

*Description of changes:* Changes from UX assurance that couldn't be done with just styling

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/StaffPayment/StaffPayment.vue [hunk 1]
```diff
@@ -59,6 +60,8 @@
           <!-- No Fee radio button -->
           <v-radio id="no-fee-radio" label="No Fee" :value="StaffPaymentOptions.NO_FEE" />
 
+          <hr>
```
Comment (cameron-freshworks) [inline src/components/StaffPayment/StaffPayment.vue:63]: What's the difference between a <br> and <hr>?

**Review by cameron-freshworks — COMMENTED** [review]
Just one comment about the divider, curious if the styling is different than other v-dividers we've used elsewhere in the project.

File: src/components/StaffPayment/StaffPayment.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/StaffPayment/StaffPayment.vue:63]: It looks to me like they're different.
https://www.w3schools.com/howto/howto_css_style_hr.asp
https://vuetifyjs.com/en/components/dividers/

Therefore I think we should use `v-divider` here for consistency.

**Review by severinbeauvais — COMMENTED** [review]
In additional to my comment above, Cam or I will need to pick this up as John has left now.

**Comment by cameron-freshworks** [comment]
> In additional to my comment above, Cam or I will need to pick this up as John has left now.

kk, no prob. I'll pull these branches and support them through!

**Comment by severinbeauvais** [comment]
> > In additional to my comment above, Cam or I will need to pick this up as John has left now.
> 
> kk, no prob. I'll pull these branches and support them through!

Thanks. If you get busy with something else, lmk and I can take it on.

**Comment by cameron-freshworks** [comment]
Closing this PR due to the work done in PR https://github.com/bcgov/bcrs-shared-components/pull/54

**Pull request closed without merging**